package.json

```diff
@@ -39,8 +40,7 @@
     "@storybook/storybook-deployer": "^2.8.1",
     "babel-loader": "^8.0.5",
     "css-loader": "^1.0.1",
-    "webpack": "^4.28.3",
-    "webpack-cli": "^3.2.3"
+    "webpack": "^4.28.3"
```

Comment:
We might not need this `webpack` dependency anymore.

Reply:
I tried the same command before trying `webpack` build command. `DE` still complained about not being able to import components. May be i didn't export it properly 😕 .

Reply:
Yeah, I think the main issue before was an `export default { ... }` instead of just `export { ... }` from `src/components/index.js`. We can switch back to `webpack` instead of `babel` if we want.